general.md

```diff
@@ -30,12 +31,19 @@
   * Document an app's environment variables in its Git repo's README.
 * Modularise and package distinct & reusable functions into libraries, i.e. Ruby
   gems or Java jars
-
-### 1.4. Accessibility
+  
+### 1.4. Selecting third-party libraries and frameworks
+Prefer software that:
+* Is open-source
```

Comment:
Is this really just a preference and not a hard requirement?

Reply:
We were reluctant with making this a strict rule because there are exceptions. For example the Zoho CRM that we are using is not open-source. We could rephrase to "We have a strong preference for software that..."

Reply:
Indeed, but this point is regarding third-party libraries and frameworks, which need to be licensed as compatible with the EUPL, as noted earlier in the document.

Reply:
That's true. I see now the "Is open-source" is already covered by the "must be compatible with EUPL" in the first section, so I propose to simply remove "Is open-source" line from section 1.4. Do you agree with that?

Reply:
Agreed, that makes sense.

Reply:
Line is removed